docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Bring your own cloud networking and security
+sidebar_label: Networking & security
+---
+
+Aiven combines a multi-cloud strategy with a cloud-agnostic approach to make the [bring your own cloud (BYOC)](/docs/platform/concepts/byoc) experience not only versatile and cost-efficient but also secure.
+
+## Bastion proxy networking
+
+The [bastion](https://en.wikipedia.org/wiki/Bastion_host) proxy service acts as a trusted
+relay between the Aiven Management plane and your BYOC environment. A typical BYOC
+implementation results in workloads that are not directly accessible from external networks.
+The bastion proxy is located in a separate virtual private cloud (VPC) and facilitates the
+communication between Aiven Management and monitoring systems and your private workloads.
+The bastion proxy service is also responsible for facilitating a communication channel that
+allows the Aiven Management plane to launch services within your private cloud.
+
+### Traffic to the bastion
+
+Aiven leverages redundant gateways to conduct BYOC operations. Ingress traffic from
+particular Aiven gateway IP addresses to the bastion subnet on specific ports is required.
+Each gateway is hosted on a separate and geographically remote cloud provider to protect
+against communication failures.
+
+| Source                 | Destination    | Description                                                                                                                                   |
+| ---------------------- | ---------------| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aiven Management plane | Bastion subnet | Aiven uses SSH from the Aiven Management plane to the bastion service node for troubleshooting, setup, or configuration tasks on the bastion. |
+| Aiven Management plane | Bastion subnet | Proxy service is used to tunnel the traffic from the Aiven Management plane to workload nodes.                                                |
+| Aiven Management plane | Bastion subnet | Aiven Management plane accesses the bastion service node to retrieve status information and perform routine operations.                       |
```

Comment:
Shouldn't this table include the ports we require?

Reply:
Kevin Fox has a deep dive doc that lists all the ports and IPs, we can exclude the ports in the public version